shared/index.d.ts

```diff
@@ -955,12 +955,94 @@ declare interface UICanvas extends Panel {
 	 */
 	DrawLinePoints(count: number, coords: number[], thickness: number, color: string): void;
 
+	/**
+	 * Draws a line, but softer!
+	 * @param count Number of points
+	 * @param coords Array of x/y coordinates. Must be count * 2 in length
+	 * @param thickness Thickness of the line
+	 * @param softness Softness of the line
+	 * @param color Color of the line
+	 */
+	DrawSoftLinePoints(count: number, coords: number[], thickness: number, softness: number, color: string): void;
```

Comment:
If it isn't much effort, specific number types would be nice to have. It isn't important, but it would give a small hint to developers for what kinds of coordinates it expects. (ex. float, i32, ...)
(This comment is for all of the canvas functions)